### PR TITLE
Feature/active inactive user

### PR DESCRIPTION
**Pull request by BobbySandhu**
- [x] I have reviewed the pull request myself. 
- [x] I confirm that code style is correct and follows CrownStack code style guide.
- [x] I have checked the code after running on Simulator/emulator/Device (which ever applicable).
- [x] I am convinced that I have not added new bugs. 
- [x] The UI behaviour is correct.

### CODE REVIEW

**Comment by BobbySandhu** [comment]
@aashishdhawan @ashutoshsingh33 please review

File: app/src/main/java/com/eqdepot/activities/UsersActivity.java [hunk 1]
```diff
@@ -159,6 +159,12 @@ public void onSuccess(Object response) {
                 MyAccountUserListData myAccountUserListData = (MyAccountUserListData) response;
                 if (myAccountUserListData != null && myAccountUserListData.getData() != null)
                     handleMyAccountResponse(myAccountUserListData.getData());
+
+                if (mAddUserAdapter.isPrimaryUser()){
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/activities/UsersActivity.java]: can be convert into ternary operator fabAddUser.setVisibility(mAddUserAdapter.isPrimaryUser() ? View.VISIBLE : View.GONE);

File: app/src/main/java/com/eqdepot/activities/AddUserActivity.java [hunk 2]
```diff
@@ -84,11 +86,14 @@ private void addUser(String firstName, String lastName, String email, String pho
...
-        DataManager.getInstance().addUser(addUserRequest, new DataManager.DataManagerListener() {
+        Session session = AppUtils.getInstance().getSessionId(this);
+
+        DataManager.getInstance().addUser(this, session.getSessionId(), addUserRequest, new DataManager.DataManagerListener() {
             @Override
             public void onSuccess(Object response) {
                 ProgressHelper.stop();
-                String message = ((AddUserResponse) response).getData().get(0).getMessage();
+                //String message = ((AddUserResponse) response).getData().get(0).getMessage();
+                String message = "Contact added successfully. An invitation link has been sent to the sepcified email.";
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/activities/AddUserActivity.java]: Please add in strings file.

File: app/src/main/java/com/eqdepot/activities/ViewUserActivity.java [hunk 3]
```diff
@@ -76,17 +88,78 @@ private void setFont() {
 
     private void getIntentData() {
         if (getIntent().hasExtra(USER_DATA_MYACCOUNT)) {
-            MyAccountUserData mMyAccountUserData = (MyAccountUserData) getIntent().getSerializableExtra(USER_DATA_MYACCOUNT);
+            mMyAccountUserData = (MyAccountUserData) getIntent().getSerializableExtra(USER_DATA_MYACCOUNT);
+            String completeName = mMyAccountUserData.getFirstName() + " " +mMyAccountUserData.getLastName();
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/activities/ViewUserActivity.java]: code Style issue.

File: app/src/main/java/com/eqdepot/activities/ViewUserActivity.java [hunk 4]
```diff
@@ -76,17 +88,78 @@ private void setFont() {
...
+        }
 
         editTextFirstName.setText(data.getFirstName());
         getEditTextLastName.setText(data.getLastName());
         editTextEmail.setText(data.getEmail());
         editTextPhone.setText(data.getPhoneMobile());
+
+        changeButtonAccess();
+    }
+
```
Comment (ashutoshsingh33) [inline app/src/main/java/com/eqdepot/activities/ViewUserActivity.java]: code style issue.

File: app/src/main/java/com/eqdepot/activities/UsersActivity.java (hunk 1 above)
Comment (BobbySandhu) [reply app/src/main/java/com/eqdepot/activities/UsersActivity.java]: done

File: app/src/main/java/com/eqdepot/activities/AddUserActivity.java (hunk 2 above)
Comment (BobbySandhu) [reply app/src/main/java/com/eqdepot/activities/AddUserActivity.java]: done

File: app/src/main/java/com/eqdepot/activities/ViewUserActivity.java (hunk 4 above)
Comment (BobbySandhu) [reply app/src/main/java/com/eqdepot/activities/ViewUserActivity.java]: done

**Pull request merged**